src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityDataBank.java

```diff
@@ -195,6 +201,10 @@ public ICubeRenderer getBaseTexture(IMultiblockPart sourcePart) {
     @Override
     public void renderMetaTileEntity(CCRenderState renderState, Matrix4 translation, IVertexOperation[] pipeline) {
         super.renderMetaTileEntity(renderState, translation, pipeline);
+        renderTextures(renderState, translation, pipeline);
+    }
+
+    protected void renderTextures(CCRenderState renderState, Matrix4 translation, IVertexOperation[] pipeline) {
```

Comment:
Why this change? Overriders could just override `renderMTE` as usual

Reply:
So that a subclass (Network Switch) could override this properly without also skipping on the super class's base texture rendering